scm/Dockerfile

```diff
@@ -26,3 +26,4 @@ RUN Install-Git -Version $env:GIT_VERSION;
 #--------------------------------------------------------------------
 
 RUN git config --system core.longpaths true
+RUN git config --system core.autocrlf false
```

Comment:
I’m thinking this should probably be in the buildbot-worker image only since we can use these images to build other things and the issue is specific to the bots.

Reply:
Good point, will update

Reply:
Moved